addons/hr_homeworking_calendar/static/tests/calendar_homeworking_tests.js

```diff
@@ -203,9 +212,10 @@ QUnit.module("homeworking", ({ beforeEach }) => {
                             type: "many2one",
                             relation: "partner",
                         },
+                        can_edit: { string: "User can edit his own record", type: "boolean"},
                     },
                     records: [
-                        { id: uid, display_name: "user 1", partner_id: 1 },
+                        { id: 1, display_name: "user 1", partner_id: 1 , can_edit: true},
```

Comment:
why did u change that? huh?
![282fe1d7e40e216c723740b0af255952](https://github.com/odoo/odoo/assets/63317845/b87729c0-324c-4543-86bf-a472fbf5203b)